docs/sql-data-sources-parquet.md

```diff
@@ -616,14 +616,15 @@ Configuration of Parquet can be done via `spark.conf.set` or by running
   <td>3.3.0</td>
 </tr>
 <tr>
-  <td><code>spark.sql.parquet.timestampNTZ.enabled</code></td>
```

Comment:
So, previous this was no-op, @gengliangwang ?

Reply:
Yes